src/ni/vsbuild/Pipeline.groovy

```diff
@@ -122,16 +122,23 @@ class Pipeline implements Serializable {
    }
 
    private void setup(lvVersion) {
+      def manifest = script.readJSON text: '{}'
+
       script.stage("Checkout_$lvVersion") {
          script.deleteDir()
          script.echo 'Attempting to get source from repo.'
          script.timeout(time: 5, unit: 'MINUTES'){
-            script.checkout(script.scm)
+            manifest['scm'] = script.checkout(script.scm)
          }
       }
       script.stage("Setup_$lvVersion") {
          script.cloneBuildTools()
          script.buildSetup(lvVersion)
+
+         // Write a manifest
+         def file = 'Built/installer/manifest.json'
```

Comment:
I would prefer this path be a constant in the Pipeline class like `build.json` is, in case we decide we want to relocate it at some point.

I'm also not crazy about requiring the output be named `Built`, but we can work that out later. `Built` should represent the directory to be archived, which is specified in `build.toml`, but for now this works.

Reply:
I considered digging into the archive parameters and pulling out location, but that felt even worse. Is there another way I should be building this path up?

Reply:
I feel like at one point I did something similar and pulled the archive information from somewhere else, but maybe I took it out because I can't find it. I think it's ok to pull that information because, ultimately, we want this file to be archived, so the base location should be archive `build_output_dir`.

That said, all of our things right now use Built, so if it's a constant path that's not buried inside a private method, I'm fine with it.

Reply:
Found it [here](https://github.com/ni/niveristand-custom-device-build-tools/blob/106dc2cadb969b2730fb302886f03c8c0635830d/src/ni/vsbuild/steps/LvBuildStep.groovy#L52). In order to get multiple files with the same name to the correct output locations, the `Build` stage looks at the archive location sometimes.

This should be fixed by creating build outputs to target-specific folders, which the new template does, but the Scan Engine stuff hasn't been updated to do that yet. Once it is, the checking of archive code should be removed from the `Build` stage